test/integration/pkg_cases.go

```diff
@@ -82,6 +82,12 @@ var cases = []struct {
 		pkgLanguage: pkg.Python,
 		pkgInfo: map[string]string{
 			"flask": "4.0.0",
+			"python-dateutil": "2.8.1",
```

Comment:
I think this file is causing the lint check to fail. There is a `make lint-fix` that should get this solved though

Reply:
Ahh, spacing was off.